enterprise/app/filter/filter.tsx

```diff
@@ -99,15 +130,50 @@ export default class FilterComponent extends React.Component<FilterProps, State>
     const startDateParam = this.props.search.get(START_DATE_PARAM_NAME);
     const endDateParam = this.props.search.get(END_DATE_PARAM_NAME);
 
-    const startDate = (startDateParam ? moment(startDateParam) : moment().subtract(7, "days")).toDate();
-    const endDate = (endDateParam ? moment(endDateParam) : moment()).toDate();
+    const defaultStartDate = getDefaultStartDateForPage(this.props.path);
+    const startDate = startDateParam ? moment(startDateParam).toDate() : defaultStartDate;
+    const endDate = endDateParam ? moment(endDateParam).toDate() : new Date();
 
     const roleValue = this.props.search.get(ROLE_PARAM_NAME) || "";
     const statusValue = this.props.search.get(STATUS_PARAM_NAME) || "";
 
     const isFiltering = Boolean(roleValue || statusValue);
     const selectedStatuses = new Set(parseStatusParam(statusValue));
 
+    const now = new Date();
+    const presetDateRanges: PresetRange[] = [
+      {
+        label: `Default (${formatDateRange(defaultStartDate, now, { now })})`,
```

Comment:
Do people need to know or care about the "default" date range?

Can we just default them to last 30 days and then they can pick whatever date range they want?

Reply:
Done -- the intent here was just to be able to "clear" the filter, but now that we can express relative date ranges in the URL, I think the ability to clear isn't as important.